browser/src/neovim/NeovimInstance.ts

```diff
@@ -701,6 +690,22 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
         }
     }
 
+    private _dispatchScrollEvent(): void {
+        if (this._pendingScrollTimeout || this._isDisposed) {
+            return
+        }
+
+        this._pendingScrollTimeout = window.setTimeout(async () => {
+            if (this._isDisposed) {
+                return
+            }
+
```

Comment:
I was also logging here so I thought this bit being put into the the event loop rather than running synchronously was where things were going missing since if by some quirk of fate if the next scroll event came straight after and the pendingScrollTimeout still existed the function would just return so it might in effect be swallowing scroll events, didn't actually verify that this was happening

Reply:
👍 Makes sense thanks for explaining